### ISSUE TITLE
Condutor: integration tests building is failed

### ISSUE THREAD

**Issue by MurashovVen** [0chain/blobber#572]
Logs: 
```
ben@vb:~/work/0chain/blobber$ docker.local/bin/build.blobber-integration-tests.sh
73f9b736da71bf8596c4cfca53dff2f708c3a8cb
Sending build context to Docker daemon  2.247MB
Step 1/16 : FROM blobber_base as validator_build
 ---> 7c02f2178198
Step 2/16 : LABEL zchain="validator"
 ---> Running in 5f894daf4c71
Removing intermediate container 5f894daf4c71
 ---> ad01233360be
Step 3/16 : RUN apk add --update --no-cache build-base linux-headers git cmake bash perl grep
 ---> Running in e70f1ab4c6fb
fetch https://dl-cdn.alpinelinux.org/alpine/v3.14/main/x86_64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/v3.14/community/x86_64/APKINDEX.tar.gz
fetch https://mirrors.aliyun.com/alpine/v3.14/main/x86_64/APKINDEX.tar.gz
fetch https://mirrors.aliyun.com/alpine/v3.14/community/x86_64/APKINDEX.tar.gz
fetch https://sjc.edge.kernel.org/alpine/v3.14/main/x86_64/APKINDEX.tar.gz
fetch https://sjc.edge.kernel.org/alpine/v3.14/community/x86_64/APKINDEX.tar.gz
fetch https://uk.alpinelinux.org/alpine/v3.14/main/x86_64/APKINDEX.tar.gz
fetch https://uk.alpinelinux.org/alpine/v3.14/community/x86_64/APKINDEX.tar.gz
fetch https://dl-4.alpinelinux.org/alpine/v3.14/main/x86_64/APKINDEX.tar.gz
fetch https://dl-4.alpinelinux.org/alpine/v3.14/community/x86_64/APKINDEX.tar.gz
fetch https://mirror.yandex.ru/alpine/v3.14/main/x86_64/APKINDEX.tar.gz
fetch https://mirror.yandex.ru/alpine/v3.14/community/x86_64/APKINDEX.tar.gz
WARNING: Ignoring https://mirror.yandex.ru/alpine/v3.14/main: No such file or directory
WARNING: Ignoring https://mirror.yandex.ru/alpine/v3.14/community: No such file or directory
OK: 311 MiB in 65 packages
Removing intermediate container e70f1ab4c6fb
 ---> 0a7f14c0caa5
Step 4/16 : ENV SRC_DIR=/blobber
 ---> Running in 6efb7692b022
Removing intermediate container 6efb7692b022
 ---> 228c0f96cc37
Step 5/16 : ENV GO111MODULE=on
 ---> Running in a88fb52fec63
Removing intermediate container a88fb52fec63
 ---> 1bb0054be10b
Step 6/16 : COPY ./go.mod          ./go.sum          $SRC_DIR/
 ---> cecd48bd2bce
Step 7/16 : RUN cd $SRC_DIR && go mod download
 ---> Running in 1f6c278a85ed
Removing intermediate container 1f6c278a85ed
 ---> dba3e481deb1
Step 8/16 : ADD . $SRC_DIR
 ---> 90fd4bb54c93
Step 9/16 : WORKDIR $SRC_DIR/code/go/0chain.net/validator
 ---> Running in 0b5fc301fd5e
Removing intermediate container 0b5fc301fd5e
 ---> 87f1a5cc8044
Step 10/16 : RUN CGO_ENABLED=1 go build -v -tags "bn256 development" -ldflags "-X github.com/0chain/blobber/code/go/0chain.net/core/build.BuildTag=$GIT_COMMIT"
 ---> Running in 39c42ba634fd
github.com/0chain/blobber/code/go/0chain.net/core/build
golang.org/x/sys/internal/unsafeheader
github.com/0chain/gosdk/core/version
github.com/0chain/blobber/code/go/0chain.net/core/config
github.com/spf13/viper/internal/encoding
golang.org/x/text/transform
github.com/hashicorp/hcl/hcl/strconv
golang.org/x/sys/unix
github.com/spf13/afero/mem
github.com/spf13/jwalterweatherman
github.com/spf13/cast
github.com/hashicorp/hcl/hcl/token
github.com/spf13/viper/internal/encoding/json
github.com/magiconair/properties
github.com/mitchellh/mapstructure
golang.org/x/text/unicode/norm
github.com/spf13/pflag
github.com/hashicorp/hcl/hcl/ast
github.com/hashicorp/hcl/hcl/scanner
github.com/hashicorp/hcl/json/token
github.com/hashicorp/hcl/json/scanner
github.com/hashicorp/hcl/hcl/parser
github.com/hashicorp/hcl/json/parser
github.com/pelletier/go-toml
github.com/hashicorp/hcl/hcl/printer
github.com/hashicorp/hcl
gopkg.in/yaml.v2
github.com/subosito/gotenv
gopkg.in/ini.v1
github.com/spf13/afero
github.com/spf13/viper/internal/encoding/hcl
go.uber.org/atomic
go.uber.org/zap/buffer
go.uber.org/multierr
go.uber.org/zap/internal/bufferpool
go.uber.org/zap/internal/color
go.uber.org/zap/internal/exit
gopkg.in/natefinch/lumberjack.v2
github.com/fsnotify/fsnotify
go.uber.org/zap/zapcore
github.com/didip/tollbooth/v6/errors
github.com/didip/tollbooth/v6/libstring
github.com/pkg/errors
golang.org/x/time/rate
github.com/andres-erbsen/clock
github.com/google/uuid
github.com/go-pkgz/expirable-cache
github.com/spf13/viper/internal/encoding/toml
go.uber.org/ratelimit
golang.org/x/crypto/sha3
github.com/tyler-smith/go-bip39/wordlists
github.com/herumi/bls-go-binary/bls
golang.org/x/crypto/pbkdf2
golang.org/x/crypto/ed25519
github.com/tyler-smith/go-bip39
github.com/0chain/gosdk/constants
github.com/lithammer/shortuuid/v3
github.com/didip/tollbooth/v6/limiter
github.com/0chain/gosdk/core/resty
github.com/spf13/viper/internal/encoding/yaml
github.com/spf13/viper
github.com/0chain/gosdk/core/logger
github.com/0chain/gosdk/zboxcore/blockchain
github.com/0chain/gosdk/core/encryption
github.com/h2non/filetype/matchers/isobmff
github.com/h2non/filetype/types
github.com/didip/tollbooth/v6
github.com/0chain/gosdk/zboxcore/logger
github.com/klauspost/cpuid/v2
go.dedis.ch/kyber/v3
github.com/h2non/filetype/matchers
go.dedis.ch/fixbuf
golang.org/x/sys/cpu
go.uber.org/zap
go.dedis.ch/kyber/v3/group/internal/marshalling
github.com/mitchellh/go-homedir
github.com/rogpeppe/go-internal/lockedfile/internal/filelock
github.com/klauspost/reedsolomon
github.com/h2non/filetype
github.com/btcsuite/btcd/chaincfg/chainhash
golang.org/x/crypto/blake2b
github.com/rogpeppe/go-internal/lockedfile
github.com/btcsuite/btcd/wire
github.com/btcsuite/btcutil/base58
github.com/btcsuite/btcd/btcec
github.com/btcsuite/btcutil/bech32
golang.org/x/crypto/ripemd160
go.dedis.ch/kyber/v3/xof/blake2xb
github.com/ethereum/go-ethereum/common/hexutil
github.com/ethereum/go-ethereum/common/math
go.dedis.ch/kyber/v3/util/random
go.dedis.ch/kyber/v3/group/mod
github.com/ethereum/go-ethereum/crypto/secp256k1
github.com/ethereum/go-ethereum/common
github.com/btcsuite/btcd/chaincfg
go.dedis.ch/kyber/v3/group/edwards25519
github.com/0chain/blobber/code/go/0chain.net/core/logging
github.com/0chain/errors
github.com/ethereum/go-ethereum/rlp
github.com/0chain/gosdk/core/util
github.com/0chain/blobber/code/go/0chain.net/core/common
github.com/0chain/gosdk/core/common
github.com/0chain/gosdk/core/conf
github.com/0chain/gosdk/zboxcore/encoder
github.com/0chain/gosdk/zboxcore/fileref
github.com/btcsuite/btcutil
github.com/0chain/gosdk/zboxcore/encryption
github.com/0chain/gosdk/core/transaction
github.com/0chain/blobber/code/go/0chain.net/core/chain
github.com/ethereum/go-ethereum/params
github.com/ethereum/go-ethereum/common/mclock
github.com/deckarep/golang-set
github.com/go-stack/stack
github.com/0chain/gosdk/core/block
github.com/btcsuite/btcutil/hdkeychain
github.com/shirou/gopsutil/internal/common
github.com/ethereum/go-ethereum/log
github.com/tklauser/numcpus
github.com/ethereum/go-ethereum/event
github.com/ethereum/go-ethereum/p2p/netutil
github.com/gorilla/websocket
github.com/tklauser/go-sysconf
github.com/0chain/blobber/code/go/0chain.net/validatorcore/config
gopkg.in/mgo.v2/internal/json
gopkg.in/mgo.v2/internal/scram
github.com/gorilla/mux
github.com/felixge/httpsnoop
github.com/shirou/gopsutil/cpu
github.com/ethereum/go-ethereum/metrics
github.com/gorilla/handlers
gopkg.in/mgo.v2/bson
github.com/ethereum/go-ethereum/rpc
gopkg.in/mgo.v2
github.com/koding/cache
github.com/0chain/blobber/code/go/0chain.net/core/cache
github.com/0chain/gosdk/core/zcncrypto
github.com/0chain/blobber/code/go/0chain.net/core/encryption
github.com/0chain/blobber/code/go/0chain.net/core/node
github.com/0chain/gosdk/zboxcore/client
github.com/0chain/gosdk/zboxcore/zboxutil
github.com/0chain/gosdk/zboxcore/marker
github.com/0chain/blobber/code/go/0chain.net/validatorcore/storage/writemarker
github.com/0chain/blobber/code/go/0chain.net/core/util
github.com/0chain/gosdk/zboxcore/allocationchange
github.com/0chain/gosdk/zboxcore/sdk
github.com/ethereum/go-ethereum/crypto
github.com/ethereum/go-ethereum/core/types
github.com/ethereum/go-ethereum
github.com/ethereum/go-ethereum/accounts
github.com/ethereum/go-ethereum/ethclient
github.com/0chain/gosdk/zcncore/ethhdwallet
github.com/0chain/gosdk/zcncore
github.com/0chain/blobber/code/go/0chain.net/core/transaction
github.com/0chain/blobber/code/go/0chain.net/validatorcore/storage
github.com/0chain/blobber/code/go/0chain.net/validator
Removing intermediate container 39c42ba634fd
 ---> 73c7541a4728
Step 11/16 : FROM golang:1.17.1-alpine3.14
 ---> da4596da1948
Step 12/16 : RUN apk add gmp gmp-dev openssl-dev
 ---> Running in ca6890511358
fetch https://dl-cdn.alpinelinux.org/alpine/v3.14/main/x86_64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/v3.14/community/x86_64/APKINDEX.tar.gz
(1/7) Installing gmp (6.2.1-r0)
(2/7) Installing libgcc (10.3.1_git20210424-r2)
(3/7) Installing libstdc++ (10.3.1_git20210424-r2)
(4/7) Installing libgmpxx (6.2.1-r0)
(5/7) Installing pkgconf (1.7.4-r0)
(6/7) Installing gmp-dev (6.2.1-r0)
(7/7) Installing openssl-dev (1.1.1l-r0)
Executing busybox-1.33.1-r3.trigger
OK: 12 MiB in 22 packages
Removing intermediate container ca6890511358
 ---> e5900637c520
Step 13/16 : COPY --from=validator_build  /usr/local/lib/libmcl*.so                         /usr/local/lib/libbls*.so                         /usr/local/lib/
 ---> fe4acc1d07ee
Step 14/16 : ENV APP_DIR=/blobber
 ---> Running in 1cc2e5ff4121
Removing intermediate container 1cc2e5ff4121
 ---> 919ed9a4219d
Step 15/16 : WORKDIR $APP_DIR
 ---> Running in d498176d8c69
Removing intermediate container d498176d8c69
 ---> 7c9e374f3655
Step 16/16 : COPY --from=validator_build $APP_DIR/code/go/0chain.net/validator/validator $APP_DIR/bin/validator
 ---> 9ad2fc74d9c3
[Warning] One or more build-args [GIT_COMMIT] were not consumed
Successfully built 9ad2fc74d9c3
Successfully tagged validator:latest
Sending build context to Docker daemon  2.247MB
Step 1/15 : FROM blobber_base as blobber_build
 ---> 7c02f2178198
Step 2/15 : RUN apk add --update --no-cache build-base linux-headers git cmake bash perl grep
 ---> Running in ab6b866f9c11
fetch https://dl-cdn.alpinelinux.org/alpine/v3.14/main/x86_64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/v3.14/community/x86_64/APKINDEX.tar.gz
fetch https://mirrors.aliyun.com/alpine/v3.14/main/x86_64/APKINDEX.tar.gz
fetch https://mirrors.aliyun.com/alpine/v3.14/community/x86_64/APKINDEX.tar.gz
fetch https://sjc.edge.kernel.org/alpine/v3.14/main/x86_64/APKINDEX.tar.gz
fetch https://sjc.edge.kernel.org/alpine/v3.14/community/x86_64/APKINDEX.tar.gz
fetch https://uk.alpinelinux.org/alpine/v3.14/main/x86_64/APKINDEX.tar.gz
fetch https://uk.alpinelinux.org/alpine/v3.14/community/x86_64/APKINDEX.tar.gz
fetch https://dl-4.alpinelinux.org/alpine/v3.14/main/x86_64/APKINDEX.tar.gz
fetch https://dl-4.alpinelinux.org/alpine/v3.14/community/x86_64/APKINDEX.tar.gz
fetch https://mirror.yandex.ru/alpine/v3.14/main/x86_64/APKINDEX.tar.gz
WARNING: Ignoring https://mirror.yandex.ru/alpine/v3.14/main: No such file or directory
fetch https://mirror.yandex.ru/alpine/v3.14/community/x86_64/APKINDEX.tar.gz
WARNING: Ignoring https://mirror.yandex.ru/alpine/v3.14/community: No such file or directory
OK: 311 MiB in 65 packages
Removing intermediate container ab6b866f9c11
 ---> cb6d18ba8db8
Step 3/15 : ENV SRC_DIR=/blobber
 ---> Running in e6bd9c136d17
Removing intermediate container e6bd9c136d17
 ---> b7f454bcf756
Step 4/15 : ENV GO111MODULE=on
 ---> Running in 41fe78181f24
Removing intermediate container 41fe78181f24
 ---> 6b5b46b7cd38
Step 5/15 : COPY ./code/go/0chain.net/go.mod          ./code/go/0chain.net/go.sum          $SRC_DIR/go/0chain.net/
COPY failed: file not found in build context or excluded by .dockerignore: stat code/go/0chain.net/go.mod: file does not exist
```

**Comment by cnlangzi** [comment]
@rrrooommmaaa  please help to fix https://github.com/0chain/blobber/issues/398 and https://github.com/0chain/blobber/issues/653. thanks

**Comment by rrrooommmaaa** [comment]
This is fixed with PR #654
